modules/ingestor/src/service/format.rs

```diff
@@ -120,39 +130,59 @@ impl<'g> Format {
     }
 
     pub fn is_csaf(bytes: &[u8]) -> Result<bool, Error> {
-        Ok(masked(
+        match masked(
             key("document").and(key("csaf_version")).and(depth(2)),
             bytes,
-        )?
-        .is_some())
+        ) {
+            Ok(Some(_)) => Ok(true),
+            Err(_) | Ok(None) => Ok(false),
+        }
```

Comment:
Don't hate me, please, but now that I understand why you're eating all the Errors potentially returned from the `is_*` fn's, I'm inclined to ask you to change all their return types from `Result<bool, Error>` to `bool`.

Reply:
I did consider it, but there's places where ... it's valid JSON, it's valid CycloneDX (or SPDX) but... not a *supported* version. Which is a valid `Error` case.

```
            Ok(Some(x)) => Err(Error::UnsupportedFormat(format!(
                "CycloneDX version {x} is unsupported; try 1.3, 1.4, or 1.5"
            ))),
```

Reply:
Good point. I guess there's no harm allowing for the possibility.

Reply:
So while we catch all errors and turn them into false, some inherently non-error results can be turned into perfectly valid Errors.